x/superfluid/keeper/stake.go

```diff
@@ -524,7 +540,11 @@ func (k Keeper) IterateDelegations(ctx sdk.Context, delegator sdk.AccAddress, fn
 		}
 
 		// get osmo-equivalent token amount
-		amount := k.GetSuperfluidOSMOTokens(ctx, interim.Denom, coin.Amount)
+		amount, err := k.GetSuperfluidOSMOTokens(ctx, interim.Denom, coin.Amount)
+		if err != nil {
+			ctx.Logger().Error("failed to get osmo equivalent of token", "Denom", interim.Denom, "Amount", coin.Amount, "Error", err)
+			continue
+		}
```

Comment:
Do we want to be continuing in this case? Based on the old logic, the panic would get propagated up the call stack

Reply:
How does the logic from this error propagation differ from all the other errors in this function?